source/dub/dub.d

```diff
@@ -277,28 +277,104 @@ class Dub {
 
 		The script above can be invoked with "dub --single test.d".
 	*/
+		struct Parser {
```

Comment:
`DCommentParser` would then also make sense as a name to make it more clear what it does.